### ISSUE TITLE
command : Close | win_ser_1 | can be sent to the IDE and closes it/hangs

### ISSUE THREAD

**Issue by giznaj** [SeleniumHQ/selenium-ide#472]
## 🐛 Bug Report
The command Close | win_ser_1|  can be sent to the IDE itself and closes it or it becomes unresponsive.  It might be an issue with how the tester is writing the commands, but nevertheless, the IDE should never try to close itself.

![image](https://user-images.githubusercontent.com/5216071/47652070-4f13ba80-db5b-11e8-93a3-3061d4eead22.png)

## To Reproduce
Please run the attached .side file

## Expected behavior
Selenium IDE should not be listening for the command and should return an error similar to "Window not found"

## Project file reproducing this issue (highly encouraged)
See attached.  Please run the script.  When the print page opens from the IDE running the command, you will have to manually close that window, then give control back to the IDE. 

## Environment
OS: Windoze 7
Selenium IDE Version:  3.4.4
Selenium SIDE Runner Version:  N/A
Node version: N/A
Browser: Chrome 
Browser Version: 69.0.3497.92 (Official Build) (32-bit)

[close.ide.zip](https://github.com/SeleniumHQ/selenium-ide/files/2525356/close.ide.zip)

**Comment by corevo** [comment]
This issue is not reproduced on my end, the closed window is the popup that is opened.  
Please explain further.

**Comment by giznaj** [comment]
Hi @corevo 
I misled you on one command.  If you run the script that is attached to the issue, close the 2 popup windows manually with the mouse.  That will leave you with the IDE and the initial browser window.  When the IDE gets to the command, it will attempt to close the IDE.  

If needed, increase the pause I put in there from 2 seconds to 10... that will allow you to close the print dialog box (popup #1) and close the popup that was to be printed (popup #2).  

In this case, you'll be left with the IDE and the first browser window.  When the close command is reached, the IDE will try and close itself.

**Comment by corevo** [comment]
I've pushed a partial fix, the IDE will never try to close itself, it just doesn't handle it very well when you try to switch to a window that you closed before hand.  
I've added a fix for that, the IDE will abort a test case when trying to switch to a window that doesn't exist.  
Later on I'll add a more extensive fix that will abort the test case immediately when the window under test is closed prematurely by the user.
